pkg/dashboard/assets/css/main.css

```
@@ -492,6 +492,10 @@ COMPONENTS
   padding-top: var(--gap-0);
 }
 
+.banner img {
+  width: 750px;
```

As-is, this won't respond to different screen sizes very well. Can we do something more like: 

```suggestion
  max-width: 750px;
  height: auto;
  object-fit: contain;
```